packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.md

```diff
@@ -131,6 +131,14 @@ a ?? (b && c && d);
 
 **_NOTE:_** Errors for this specific case will be presented as suggestions (see below), instead of fixes. This is because it is not always safe to automatically convert `||` to `??` within a mixed logical expression, as we cannot tell the intended precedence of the operator. Note that by design, `??` requires parentheses when used with `&&` or `||` in the same expression.
 
+### `ignorePrimitives`
+
+If you would like to ignore certain primitive types that can be falsy then you may pass an object containing a boolean value for each primitive:
+
+- `string: true`, ignores `null` or `undefined` unions with `string` (default: false).
+- `number: true`, ignores `null` or `undefined` unions with `number` (default: false).
```

Comment:
@JoshuaKGoldberg Should I add an option for bigint? I realized that `!0n === true` only now

Reply:
If you have the time, sure! I don't think very many people are working with bigints, but it's a nice nice-to-have.

Reply:
Done